README.md

```diff
@@ -73,15 +73,20 @@ _Note that the first time you wish to run the commit analyzer, you'll have to ru
 
 ### Merging an upstream repository into your fork
 
+If your fork and branch are behind by a number of commits (meaning there have been a lot of changes made to the master branch since you originally forked it), you may need to update your fork to reflect the latest changes. This involves sending a pull request that asks for all changes in the current master branch to be merged into your fork and branch. This is the opposite of a pull request that you would normally send (which merges your changes into the CenturyLinkCloud/PublicKB master branch).
+
 **Via Github.com Website**
 
-1. Open your fork on GitHub.
-2. Click on Pull Requests.
-3. Click on New Pull Request. By default, GitHub will compare the original with your fork; there shouldn’t be anything to compare if you didn’t make any changes.
-4. Click on switching the base. Now GitHub will compare your fork with the original, and you should see all the latest changes.
-5. Click on Click to create a pull request for this comparison and assign a predictable name to your pull request (e.g., Update from original).
-6. Click on Send pull request.
-7. Scroll down and click Merge pull request and finally Confirm merge. If your fork didn’t have any changes, you will be able to merge it automatically.
+1. Open your fork (and branch) on GitHub.
+2. Click the **Pull Request** button (next to the **Branch:** drop down list).
+3. Click **New Pull Request**. By default, GitHub will compare the master branch of the CenturyLinkCloud/PublicKB with your fork and branch.  
+    **Note:** If you have not made any changes to your fork yet, there shouldn’t be anything to compare.
+4. Reverse the order of the repositories and branches in the **Base Fork:** and **Head Fork:** drop down lists. Your repository and branch should come before the CenturyLinkCloud/PublicKB master repository and branch. This allows GitHub to compare your fork with the master. You should then see all the latest changes that have been made to the master branch.
+5. Click the **Pull Request** button to create a pull request for this comparison.  
+    **Note:** You should assign a predictable name to your pull request (e.g., Update from original).
+6. Click on **Send Pull Request**.
+7. Scroll down and click **Merge Pull Request** and then **Confirm Merge**. If your fork didn’t have any changes, you will be able to merge it automatically.  
+    **Note:** You will not be able to merge your own pull request unless you have write access to the CenturyLink Cloud repository.
```

Comment:
I'd say you should never merge your own PR, in general you want to wait for comment form at least one other person.